viz/charts/linkerd-viz/values.yaml

```diff
@@ -12,6 +12,8 @@ clusterDomain: cluster.local
 # @default -- clusterDomain
 identityTrustDomain: ""
 
+extensionAnnotation: linkerd.io/extension
```

Comment:
I don't think we want the annotation key to be configurable.  I think we should revert this part of the change.

Reply:
Ah ok. I can get that removed.